jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e.yaml

```diff
@@ -1034,18 +1034,56 @@
     provider-env: '{gke-provider-env}'
     jobs:
         - 'kubernetes-e2e-{provider}-{version-old}-{version-new}':
+            version-old: '1.0'
+            version-new: '1.2'
+            version-infix: '1-0-1-2'
+            version-env: |
+                # In v1.1-, NUM_MINIONS defaults to 2, so we have to override to 3 here.  NUM_MINIONS
+                # was changed to NUM_NODES in v1.2, but we don't need to override for v1.2+.
+                export NUM_MINIONS=3
+                # Similarly, in v1.1-, MINION_SIZE defaults to 'n1-standard-1', so we need to
+                # override it here (specifically for HPA tests).  MINION_SIZE was changed to
+                # NODE_SIZE in v1.2, but we don't need to override for v1.2+.
+                export MINION_SIZE='n1-standard-2'
+            legacy-ginkgo-test-args-env: |
+                # XXX This is a hack to run only the tests we care about, without importing all of
+                # the skip list vars from the v1.0 e2e.sh.
+                export GINKGO_TEST_ARGS="--ginkgo.skip=Skipped|Restart\sshould\srestart\sall\snodes|Example|Reboot|ServiceLoadBalancer|DaemonRestart\sController\sManager|Daemon\sset\sshould\srun\sand\sstop\scomplex\sdaemon|Resource\susage\sof\ssystem\scontainers|allows\sscheduling\sof\spods\son\sa\sminion\safter\sit\srejoins\sthe\scluster"
+        - 'kubernetes-e2e-{provider}-{version-old}-{version-new}':
             version-old: '1.1'
             version-new: '1.2'
             version-infix: '1-1-1-2'
             version-env: |
-                # In v1.1, NUM_MINIONS defaults to 2, so we have to override to 3 here.  NUM_MINIONS
+                # In v1.1-, NUM_MINIONS defaults to 2, so we have to override to 3 here.  NUM_MINIONS
```

Comment:
Er, v1.1 and below (as opposed to v1.2+).  If that's confusing, which I guess it is, would "and below" be clearer?